ci-operator/step-registry/openshift/microshift/rebase/openshift-microshift-rebase-ref.yaml

```diff
@@ -0,0 +1,17 @@
+ref:
+  as: openshift-microshift-rebase
+  from: src
+  cli: nightly
+  grace_period: 30m
+  commands: openshift-microshift-rebase-commands.sh
+  resources:
+    requests:
+      cpu: 100m
+      memory: 100Mi
+  dependencies:
+  - name: "release:nightly-amd64"
```

Comment:
Should this be `release:nightly`? I didn't see `release:nightly-amd64` elsewhere in the repo, but there is `release:nightly-arm64`.

Reply:
oh, I see you defined the nightly-amd64 in *main__periodics.yaml